native_client/ctcdecode/__init__.py

```diff
@@ -10,12 +10,11 @@ class Scorer(swigwrapper.Scorer):
     :type alpha: float
     :param beta: Word insertion bonus.
     :type beta: float
-    :model_path: Path to load language model.
-    :trie_path: Path to trie file.
+    :model_path: Path to load scorer.
```

Comment:
nit: `scorer_path` to be consistent everywhere ?

Reply:
Done.